app/src/main/java/org/gdg/frisbee/android/adapter/ContributorAdapter.java

```diff
@@ -1,60 +1,80 @@
+/*
+ * Copyright 2013, 2015 The GDG Frisbee Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package org.gdg.frisbee.android.adapter;
 
 import android.content.Context;
+import android.os.Build;
 import android.text.TextUtils;
-import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ArrayAdapter;
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.squareup.picasso.Picasso;
+
 import java.util.Collection;
 
 import org.gdg.frisbee.android.R;
 import org.gdg.frisbee.android.api.model.Contributor;
-import org.gdg.frisbee.android.app.App;
-
-/**
- * Created with IntelliJ IDEA.
- * User: maui
- * Date: 08.07.13
- * Time: 02:28
- * To change this template use File | Settings | File Templates.
- */
-public class ContributorAdapter extends ArrayAdapter<Contributor> {
 
-    private Context mContext;
-    private LayoutInflater mInflater;
+public class ContributorAdapter extends ArrayAdapter<Contributor> {
 
-    public ContributorAdapter(Context context, int textViewResourceId) {
-        super(context, textViewResourceId);
-        mContext = context;
-        mInflater = LayoutInflater.from(mContext);
+    public ContributorAdapter(Context context) {
+        super(context, R.layout.contributor_item);
     }
 
     @Override
-    public View getView(int position, View view, ViewGroup parent) {
-        if(view == null)
-            view = mInflater.inflate(R.layout.contributor_item, null);
+    public View getView(int position, View convertView, ViewGroup parent) {
+        if (convertView == null) {
+            convertView = View.inflate(getContext(), R.layout.contributor_item, null);
+
+            ViewHolder viewHolder = new ViewHolder();
+            viewHolder.contributorName = (TextView) convertView.findViewById(R.id.contributorName);
```

Comment:
Use ButterKnife to annotate `contributorName` field, and these could be removed.